.github/workflows/validate-pr.yml

```diff
@@ -19,14 +19,34 @@ jobs:
         with:
           node-version: 12
 
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
       - name: Cache dependencies
         uses: actions/cache@v2
         with:
-          path: /usr/local/share/.cache/yarn/v4
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

Comment:
Sorry, this one was my fault. I was hoping to skip a step by defining the cache path (assuming we always run this on an ubuntu container). Thanks for adding this back in.

Since we've got so much boilerplate, how would y'all feel about abstracting these setup actions (including that great gatsby stuff below) into a repo.

Either way, we will probably want to update the [other workflow PR](https://github.com/newrelic/docs-website/pull/552) to include this (cc: @roadlittledawn)

Reply:
No problem! Figured its probably best to let yarn tell us where it is rather than assume it.

> Since we've got so much boilerplate, how would y'all feel about abstracting these setup actions (including that great gatsby stuff below) into a repo?

While I agree the boilerplate is a bit annoying, I'm not sure it warrants an entire repo just to abstract these couple steps. Copy/paste works fine for me right now. Only reason I say that is because another repo means all the overhead of going through the open source checklist and putting it in the right org.